tap_shopify/streams/base.py

```diff
@@ -77,6 +78,10 @@ def is_timeout_error(error_raised):
     return True
 
 def shopify_error_handling(fnc):
+    @backoff.on_exception(backoff.expo, # IncompleteRead error raised
```

Comment:
Can we merge this inside below backoff?

Reply:
No, we cannot merge it as it has giveup function [here](https://github.com/singer-io/tap-shopify/pull/144/files#diff-32aec538112132532b027564fa1c51d48c6f89c0631d698d4a8b450c6668b674L82)